packages/formstr-app/src/containers/CreateFormNew/components/QuestionCard/InputElements/OptionTypes/DropdownCreator.tsx

```diff
@@ -27,33 +27,42 @@ export const DropdownCreator: React.FC<RadioButtonCreatorProps> = ({
 
   const getMenuItems = (): MenuProps["items"] => {
     return choices.map((choice) => {
-      console.log("Choice is", choice);
       let [choiceId, label, settingsString] = choice;
       let settings = JSON.parse(settingsString || "{}") as ChoiceSettings;
+      const isOtherOption = settings.isOther;
+      
       return {
         label: (
           <div className="radioButtonItem" key={choiceId}>
             <Input
-              defaultValue={label}
+              defaultValue={isOtherOption ? "Other" : label}
               key={choiceId}
               onChange={(e) => {
                 handleLabelChange(
                   e.target.value,
-                  choiceId!,
+                  choiceId,
                   choices,
                   handleNewChoices
                 );
               }}
-              placeholder="Enter an option"
+              placeholder={isOtherOption ? "Other" : "Enter an option"}
               className="choice-input"
-              disabled={settings.isOther}
+              disabled={isOtherOption}
             />
+            {isOtherOption && (
```

Comment:
let's show the input field (disabled if not selected) even if the user doesn't select it, it is very confusing when there is an `Other:` option but we don't see the input field until we select it